packages/gatekeeper/src/gatekeeper-client.ts

```diff
@@ -146,14 +175,18 @@ export default class GatekeeperClient {
         }
     }
 
-    createDID(operation) {
-        return this.handleDIDOperation(operation);
+    async createDID(operation: Operation): Promise<string> {
+        const result = await this.handleDIDOperation(operation);
+        if (typeof result !== 'string') {
+            throw new Error(`Expected string from createDID, got ${typeof result}`);
```

Comment:
Why is this necessary?

Reply:
Return type of handleDIDOperation is `string | boolean`, but createDID only returns a string. If someone calls createDID with the type set to `update` then `handleDIDOperation` would return a `boolean`. Typescript will not just let us return the result of `handleDIDOperation` as it that does not match the return type of createDID so we need to rule out unwanted return types.